logfire-api/logfire_api/_internal/integrations/system_metrics.pyi

```diff
@@ -0,0 +1,16 @@
+from _typeshed import Incomplete
+from logfire import Logfire as Logfire
+from typing import Iterable
+from typing_extensions import LiteralString
+
+MetricName: Incomplete
```

Comment:
@Kludex why is this Incomplete?

Reply:
Good question. I usually try stuff, and look if there are issues on mypy repository...

If you add the `type[Literal["potato"]]` explicitly, does it work?

Reply:
sort of :/